libs/common/src/assets/i18n/es.json

```diff
@@ -293,6 +293,7 @@
 	"reset.password_reset": "Se ha restablecido la contraseña",
 	"reset.title": "Crea contraseña",
 	"resource.access_source_file": "Acceder al documento de origen",
+	"resource.authorship": "Paternidad literaria",
```

Comment:
What about change to "Autoría" ?

Reply:
no problem with me, I just took what babeledit was suggesting (it uses google translate)